# AssertionError occurs when the condition of the assert operator is not satisfied
assert 2 + 2 == 5, ("Math", "still", "works")

# assert operator is used for programmer errors,
# so you do not have to catch AssertionError
